app/manifest/v2/_base.json

```diff
@@ -64,14 +64,11 @@
     "storage",
     "unlimitedStorage",
     "clipboardWrite",
-    "http://localhost:8545/",
-    "https://*.infura.io/",
-    "https://*.codefi.network/",
-    "https://*.cx.metamask.io/",
-    "https://chainid.network/chains.json",
-    "https://lattice.gridplus.io/*",
+    "http://*/*",
+    "https://*/*",
```

Comment:
I don't believe so.

Reply:
That said, we already practically have these `allow-all` permissions via the `content_scripts` `matches`. However, content_script only works _after_ redirects have settled, so they aren't useful.

Not sure it makes it any better, but the MV3 manifest does already include the same `*` permissions:

```
  "host_permissions": [
    "http://localhost:8545/",
    "file://*/*",
    "http://*/*",
    "https://*/*"
  ],
  ```

Reply:
Due to the existing contentscript permissions, these additions don't really grant us much in terms of added capabilities. The only differences I am aware of are that it lets us use the `webRequest` API, and it disables CORS checks.

I would like to avoid adding further permissions, but, in this case it does seem like we have a good reason.